packages/fx-core/resource/aad-manifest-change-logs.md

```diff
@@ -0,0 +1,27 @@
+# Congratulations! You have successfully upgraded your Teams App project structure.
+
+We have updated the configuration files so that your project is compatible with the latest Teams Toolkit features, including add a AAD manifest template file.
+
+> Import Notes: If you collaborate on this project with your co-workers, and his teams toolkit extension version <= 3.7.0, please ensure your team members update the Teams Toolkit extension to the latest version after committing the changes with this upgrade.
+
+## Why upgrade
+Teams Toolkit continues to improve your Teams application development experience. We are upgrading the Teams app project structure so that you can:
+
+1. Use AAD manifest template to customize your AAD app.
+1. Allow to add single sign-on (SSO) feature for Bot (hosting on Azure App Service), Messaging Extension, Static Launch Page, and Embed existing web App
+
+## Know about the changes we made
+After the project upgrade, there are following changes we made:
+1. Update `.fx\configs\projectSettings.json` capabilities to include TabSSO and/or BotSSO based on original project capabilities.
+1. AAD manifest template file will be added to `templates\appPackage\aad.template.json`
+1. The required resource access information in `permissions.json` file will be merged into `aad.template.json` file
```

Comment:
Do we delete `permission.json` file?

Reply:
We do not delete permissions.json file as discussed to keep combability with extension after 4.18 release

Reply:
Can say something like this: The required resource access information in `permissions.json` file will be merged into `aad.template.json` file and `permission.json` file will be deprecated, please customize resource access in `aad.template.json` file.

Reply:
Thanks, update